dependencies.gradle

```diff
@@ -0,0 +1,11 @@
+ext.versions = [
+        detekt: "1.19.0",
+]
+
+ext.buildLibs = [
+        detektGradlePlugin: "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:$versions.detekt",
+]
+
+ext.configs = [
+        coreLibrary: "${project.rootDir}/build-configuration/core-library.gradle",
+]
```

Comment:
Starts a more structured way of declaring ext constants. The idea being to move all existing declarations here.